tests/schema/tests.py

```diff
@@ -2303,6 +2304,54 @@ class Meta:
         columns = self.column_classes(Author)
         self.assertEqual(columns["birth_year"][1].default, "1988")
 
+    @isolate_apps("schema")
+    def test_add_text_field_with_db_default(self):
+        class Author(Model):
+            description = TextField(db_default="(missing)")
+
+            class Meta:
+                app_label = "schema"
+
+        with connection.schema_editor() as editor:
+            editor.create_model(Author)
+
+    @isolate_apps("schema")
+    def test_db_default_equivalent_sql_noop(self):
+        class Author(Model):
+            name = TextField(db_default=Value("foo"))
+
+            class Meta:
+                app_label = "schema"
+
+        with connection.schema_editor() as editor:
+            editor.create_model(Author)
+
+        new_field = TextField(db_default="foo")
+        new_field.set_attributes_from_name("name")
+        new_field.model = Author
+        with connection.schema_editor() as editor, self.assertNumQueries(0):
```

Comment:
Ensures generated migrations from `db_default=Value("foo")` to `db_default="foo"` do not result in any schema alterations.